### DISCUSSION TITLE
[16] Dinder - Review

### DISCUSSION THREAD

**Discussion by Manuel-F-04** [dhbw-ka-tinf22b5-dinder/Dinder#28] [Projektblog]
## Review
[Code Review](https://github.com/dhbw-ka-tinf22b5-dinder/Dinder-SRS/blob/main/Review.md)

## Pipeline
Die Pipeline wurde im letzten Blogpost schon vollständig beschrieben.

## Projektentwicklung
- Hinzufügen von den Frontend-Seiten New_Advertisement und Swiped_Advertisements. Allerdings fehlt noch die Verbindung zum Backend.
- Behebung von Erscheinungsproblemen im Frontend (NavBar, CreateAdvertisement)
- Endpoints für hinzugefügten Frontend-Seiten
- Zusätzliche Backend-Tests
- Begebung von Code-Quality-Fehlern im Frontend

**Comment by ghost** [comment]
Schöner Projektfortschritt den ihr hier verzeichnen könnt!

Interessant zu wissen wäre, warum ihr beim Review _"die Bereiche betrachtet, die noch entwickelt werden oder einfach zu ändern sind."_ ? Mehr Potential für Sinnhaftigkeit bestünde doch darin, die Teile anzuschauen, welche bereits entwickelt wurden und kritische Komponenten hinsichtlich Funktionalität darstellen. 

Falls ihr anderer Meinung seid oder euer zitierter Satz falsch verstanden wurde, gerne berichtigen. :)

**Comment by niklaskrauth** [comment]
Gut zu wissen das ihr viele unterschiedliche Aspekte in der Review anspricht und das ihr das Backend als auch das Fronten abcheckt. Auch schön zu sehen ist das eure Review zu einem guten Ergebnis gekommen ist.
